lint check fails on master

A recent change of flake8 or its config causes the code style check to fail on the master branch - it complains about a `string.format(...)` function used  in _signing.py file:

`google/cloud/storage/_signing.py:156:13: F523 '...'.format(...) has unused arguments at position(s): 0`

/cc @frankyn

--- a/google/cloud/storage/_signing.py
+++ b/google/cloud/storage/_signing.py
@@ -153,9 +153,7 @@ def get_expiration_seconds_v4(expiration):
 
     if seconds > SEVEN_DAYS:
         raise ValueError(
-            "Max allowed expiration interval is seven days (%d seconds)".format(
-                SEVEN_DAYS
-            )
+            "Max allowed expiration interval is seven days {}".format(SEVEN_DAYS)
         )
 
     return seconds

